packages/cypress/package.json

```diff
@@ -58,7 +58,7 @@
     "knapsack-pro-cypress": "lib/knapsack-pro-cypress.js"
   },
   "dependencies": {
-    "@knapsack-pro/core": "^7.1.0",
+    "@knapsack-pro/core": "^8.0.0",
```

Comment:
So I was running into an issue where working locally, `@knapsack-pro/vitest` wasn't using the version in the monorepo.  `npm` was installing version 7.1.0, instead of linking to the monorepo version, since the version in the monorepo was 8.0.0

But sure I'll revert this